integration/localnet/bootstrap.go

```diff
@@ -331,6 +335,11 @@ func prepareService(container testnet.ContainerConfig, i int) Service {
 		},
 	}
 
+	// TODO: having trouble enabling admin tool for access node. skip Access node for now.
```

Comment:
Can you describe the error? Were there logs? It definitely works on access nodes (Access Nodes on the current mainnet has it enabled), so it's probably something that can be fixed. I might be able to help here.

Reply:
Access nodes kept crashing on startup, due to port to be used. I'm not sure which module is also using `9002`, so I changed to `9005`, it worked, but crashed again for a different reason, saying grpc server is fatal to start.